demo/JENKINS_HOME/credentials.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<com.cloudbees.plugins.credentials.SystemCredentialsProvider plugin="credentials@1.9.4">
-  <domainCredentialsMap class="hudson.util.CopyOnWriteMap$Hash">
-    <entry>
-      <com.cloudbees.plugins.credentials.domains.Domain>
-        <specifications/>
-      </com.cloudbees.plugins.credentials.domains.Domain>
-      <java.util.concurrent.CopyOnWriteArrayList>
-        <com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl>
-          <scope>GLOBAL</scope>
-          <id>jenkins-login</id>
-          <description></description>
-          <username>jenkins</username>
-          <password>jenkins</password>
-        </com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl>
-      </java.util.concurrent.CopyOnWriteArrayList>
-    </entry>
-  </domainCredentialsMap>
-</com.cloudbees.plugins.credentials.SystemCredentialsProvider>
```

Comment:
Right, no replacement is needed, because this was just credentials for the `SSHLauncher` formerly used in `config.xml`.